adapters/bombcrypto-bnb/index.js

```diff
@@ -0,0 +1,148 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const Binance = require("../../sdk/binance");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const BigNumber = require("bignumber.js");
+
+class BombCrypto {
+    constructor() {
+        this.name = "bombcrypto-bnb";
+        this.symbol = "BNB";
+        this.token = "bnb";
+        this.protocol = "binance";
+        this.block = 8867059;
+        this.contract = "0xe29F0B490F0d89CA7ACAc1C7BeD2E07eCAD65201";
```

Comment:
All EVM chains (Ethereum, Polygon, BSC, ...) contracts must be lowercased

Reply:
Thanks for the review Liutauras. I have converted the Contract Address to lowercase.